### DISCUSSION TITLE
wildrift-gold-efficiency/5.2/

### DISCUSSION THREAD

**Discussion by giscus** [changchiyou/wildrift-gold-efficiency#25] [Announcements]
# wildrift-gold-efficiency/5.2/

Wild Rift Gold Efficiency of 5.2 (12 adjusted, 3 new items)

http://localhost:4000/wildrift-gold-efficiency/5.2/

**Comment by changchiyou** [comment]
[![](https://github.com/user-attachments/assets/1a5c5180-a20b-4b4e-b274-0d5785b3bc25)](https://www.reddit.com/r/wildrift/comments/1esme1a/comment/li75h0w/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button)
